cl/corpus_importer/management/commands/make_aws_manifest_from_search.py

```diff
@@ -0,0 +1,165 @@
+import csv
+import io
+from typing import Any
+
+import boto3
+from django.core.management import CommandParser  # type: ignore
+from django_elasticsearch_dsl.search import Search
+from elasticsearch_dsl import Q
+
+from cl.lib.command_utils import VerboseCommand
+from cl.lib.elasticsearch_utils import (
+    add_fields_boosting,
+    build_fulltext_query,
+    build_sort_results,
+)
+from cl.search.constants import SEARCH_OPINION_QUERY_FIELDS
+from cl.search.documents import OpinionDocument
+from cl.search.models import SEARCH_TYPES
+
+s3_client = boto3.client("s3")
+
+
+def build_base_query(options: dict[str, Any]) -> Search:
+    """
+    Constructs a base Full text Elasticsearch query using the provided options.
+
+    Args:
+        options (dict[str, Any]): A dictionary containing search options:
+            - search_text (str): The text to search for.
+            - search_type (str): The type of search.
+            - es_page_size (int): The desired number of results per page.
+
+    Returns:
+        Search: The constructed Elasticsearch query object.
+
+    Raises:
+        NotImplementedError: If the provided search_type is not supported.
+    """
+    cd = {
+        "q": options["search_text"],
+        "type": options["search_type"],
+    }
+    match options["search_type"]:
+        case SEARCH_TYPES.OPINION:
+            unique_sorting_key = "id"
+            cd["order_by"] = "score desc"
+            search_query = OpinionDocument.search()
+            search_fields = SEARCH_OPINION_QUERY_FIELDS.copy()
+            search_fields.extend(
+                add_fields_boosting(
+                    cd,
+                    [
+                        "type",
+                        "text",
+                        "caseName",
+                        "docketNumber",
+                    ],
+                ),
```

Comment:
Why do we need boosting?

Reply:
Thanks for pointing this out. We don't need to boost this query because we're not using the score. We just need to ensure including those fields in the full-text query. I'll remove the boosting method.